Sources/Widgets/Representations/ResliceCursorContextRepresentation/index.js

```diff
@@ -296,33 +311,39 @@ function vtkResliceCursorContextRepresentation(publicAPI, model) {
 // Object factory
 // ----------------------------------------------------------------------------
 
-const DEFAULT_VALUES = {
-  axis1Name: '',
-  axis2Name: '',
-  coincidentTopologyParameters: {
-    Point: {
-      factor: -1.0,
-      offset: -1.0,
-    },
-    Line: {
-      factor: -1.5,
-      offset: -1.5,
+function defaultValues(initialValues) {
+  return {
+    axis1Name: '',
+    axis2Name: '',
+    coincidentTopologyParameters: {
+      Point: {
```

Comment:
Shouldn't "Point", "Line", "Polygon" be an enum (or use an existing enum)?

Reply:
Such enum does not exist (and it is not the topic of this PR :-) )